code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -237,18 +242,16 @@
 		. = TRUE //no afterattack
 		if(machine_stat & (NOPOWER|BROKEN))
 			return
-		if(beaker)
-			to_chat(user, "<span class='warning'>A container is already loaded into [src]!</span>")
-			return
+		var/obj/item/reagent_containers/B = I
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Don't name the var `B`

Reply:
In my defense, that is exactly how the already existing chem-code handles it.
But I'll go ahead and change it to something else.
Hopefully the name I came up with is clear.

Reply:
thats fine, its just new code has to be up to the guidelines is all